rancher-agent-2.11.yaml

```
@@ -1,7 +1,7 @@
 package:
   name: rancher-agent-2.11
   version: "2.11.1"
-  epoch: 1
+  epoch: 0
```

Shouldn't the epoch go up?

```suggestion
  epoch: 2
```